style.css

```diff
@@ -0,0 +1,313 @@
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin: 12px 0;
+}
+
+.grid-item {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  background-color: #fff;
+  color: #344563;
+  background-repeat: no-repeat;
+  background-position: 16px 16px;
+  border-radius: 16px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 16px;
+  width: 327px;
+  height: 552px;
+}
+
+.grid-item ul {
+  width: 203px;
+  height: 24px;
+  padding-left: 0;
+}
+
+.content ul {
+  list-style-type: disc;
+  justify-content: space-between;
+  width: 236px;
+  padding-left: 0;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.content li {
+  color: #7a869a;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.ocontent li {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  margin-left: 16px;
+  background-color: #ebebff;
+  color: #6070ff;
+  border-radius: 8px;
+  padding: 6px 12px;
+  align-items: center;
+}
+
+.grid-item p {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  margin: 12px 16px;
+}
+
+.ttt {
+  list-style-type: none;
+}
+
+ul .ttt {
+  color: #000;
+}
+
+button {
+  width: 124px;
+  height: 48px;
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  color: #6070ff;
+  background-color: #fff;
+  margin-bottom: 28px;
+  margin-left: 16px;
+}
+
+button:hover {
+  background-color: #6070ff;
+}
+
+button:active {
+  background-color: #2230d2;
+}
+
+.show-case {
+  background-image: url("../img/Snapshoot-Portfolio1.png");
+  margin-top: 114px;
+}
+
+.show-case2 {
+  background-image: url("../img/Snapshoot-Portfolio2.png");
+  margin-top: 88px;
+}
+
+.show-case3 {
+  background-image: url("../img/Snapshoot-Portfolio3.png");
+  margin-top: 88px;
+}
+
+.show-case4 {
+  background-image: url("../img/Snapshoot-Portfolio4.png");
+  margin-top: 88px;
+}
+
+#about-me {
+  flex-direction: column;
+  width: 375px;
+  background: #fff;
+  display: flex;
+  border-top-right-radius: 80px;
+  margin-top: 114px;
+}
+
+h2 {
+  position: static;
+  width: 327px;
+  height: 101px;
+  left: 0;
+  top: 0;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 50px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 12px 24px;
+  margin-top: 114px;
+}
+
+#about-me p {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  width: 327px;
```

Comment:
Kindly, I recommend that you do not limit the texts with a fixed width, but that they adapt freely to the container